core/coreapi/unixfs.go

```diff
@@ -149,6 +149,13 @@ func (api *UnixfsAPI) Get(ctx context.Context, p path.Path, metadata bool) (file
 		return nil, err
 	}
 
+	if metadata {
+		mnd, err := ft.GetMetaSubdagRoot(ctx, nd, ses.dag)
+		if err != nil {
+			return nil, err
+		}
+		return unixfile.NewUnixfsFile(ctx, ses.dag, mnd, metadata)
```

Comment:
In my other go-unixfs PR it fixes this issue on the unixfs layer - after you review you may not need this change.

Reply:
could you check if this is still needed

Reply:
Not this one. Thank you. Steve